src/constants.js

```diff
@@ -22,3 +22,5 @@ export const ACTIONS = {
   REFRESH_NODE_LABELS: 'REFRESH_NODE_LABELS',
   SET_NODE_LIMIT: 'SET_NODE_LIMIT'
 };
+//To set initial label for type 'person' create an entry named "PERSON_LABEL"
+export const INITIAL_LABEL_MAPPINGS = { SOFTWARE_LABEL: 'lang' }
```

Comment:
Comment out the mapping and leave it as an example, but we don't actually want any initial defaults. The visualizer is intended to be usable on any generic graph out of the box. We intend to have customized versions expecting certain graph structures, but the core gremlin-visualizer shouldn't include any graph default customizations.

Reply:
i.e.
```
export const INITIAL_LABEL_MAPPINGS = { 
// SOFTWARE: 'name' 
}
```